CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## Version 1.15.0
+* Release date: June 15, 2022
+* Release status: GA
+
+### What's new in 1.15.0
+* Add support for dependent extensions
+* Fixed bugs under June Milestone
```

Comment:
It'd be good to add links to these things - not everyone is going to know where this is 

https://github.com/microsoft/vscode-mssql/issues?q=is%3Aissue+milestone%3A%22June+2022+Release%22+is%3Aclosed

I'd suggest just making this a link with a generic title too - something like `[Fixed issues](https://github.com/microsoft/vscode-mssql/issues?q=is%3Aissue+milestone%3A%22June+2022+Release%22+is%3Aclosed)`

Reply:
@erpett @Benjin